src/Core/DevOps/System/Command/SystemDumpDatabaseCommand.php

```diff
@@ -22,6 +23,13 @@ public function __construct(string $defaultDirectory, Connection $connection)
         $this->connection = $connection;
     }
 
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->addOption('ignore-tables', 'i',  InputArgument::OPTIONAL, 'Tables to ignore on export', 'enqueue,message_queue_stats,dead_message');
```

Comment:
Why not using ` InputArgument::IS_ARRAY` so you don't need to provide them comma-seperated and don't need to explode the value of the argument

Reply:
I actually tried that but then I didn't knew how to ignore no table at all :(

Reply:
You can combine both flags:
```
InputArgument::IS_ARRAY | InputArgument::OPTIONAL
```
from the symfony docs: https://symfony.com/doc/current/console/input.html#using-command-arguments

Reply:
HELLO THERE. Then I can rewrite it to that and I am good with that

Reply:
I see where my issue was ... I used `InputArgument` with an option ... now I use `InputOption` and then it works :)